.gitignore

```diff
@@ -61,3 +61,4 @@ volontulo/migrations/*
 
 # PyCharm directory
 .idea/
+migrations/
```

Comment:
That's not necessary - I belive this migration directory is old artefact, which was used before migrating to reusable app. You can freely remove it.